### PR TITLE
PLT-3855 Fixed reset password error

### PR DESCRIPTION
**Pull request by DavidLu1997**
#### Summary
Fixes wrong error message when resetting password through system console. Should see same error message as everywhere else.

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-3855

### CODE REVIEW

File: webapp/components/admin_console/reset_password_modal.jsx [hunk 1]
```diff
@@ -3,17 +3,10 @@
...
 
-import {injectIntl, intlShape, defineMessages, FormattedMessage} from 'react-intl';
-
-var holders = defineMessages({
-    submit: {
-        id: 'admin.reset_password.submit',
-        defaultMessage: 'Please enter at least {chars} characters.'
-    }
-});
+import {injectIntl, intlShape, FormattedMessage} from 'react-intl';
```
Comment (enahum) [inline webapp/components/admin_console/reset_password_modal.jsx]: it seems to me that now you can remove the `injectIntl` and `intlShape`  variables from the import

Comment (DavidLu1997) [inline webapp/components/admin_console/reset_password_modal.jsx]: Is it just me or is `injectIntl` still used at the very bottom?

**Comment by enahum** [comment]
ok maybe create a ticket to remove them later and assign that to me. thanks

+1

File: webapp/components/admin_console/reset_password_modal.jsx (hunk 1 above)
Comment (hmhealey) [inline webapp/components/admin_console/reset_password_modal.jsx]: You can remove those as well. Just remove both lines and change the class declarataion to be
```
export default class ResetPasswordModal extends React.Compoennt {
```

**Comment by hmhealey** [comment]
+1

**Comment by jasonblais** [comment]
@DavidLu1997 It's now correct in the System Console, but incorrect in the password screen

image
image

**Comment by enahum** [comment]
@jasonblais after clarification from @DavidLu1997 can you confirm this is working as expected

**Comment by DavidLu1997** [comment]
Confirmed working when I tried it
image

**Comment by jasonblais** [comment]
Sorry for the delay:

Tested on Windows 10, Chrome. After a refresh
 - correct error messages on System Console
 - correct error messages on Account Settings > Security > Password
 - correct error messages on account creation page via team invite link and the "Create one now" link on an open server

Tested account creation with email with no problems, and password requirements satisfied

**Comment by jasonblais** [comment]
+1

**Pull request merged**